dnf-plugins-core.spec

```diff
@@ -26,7 +35,8 @@ protected_packages passive plugins.
 %package -n python-dnf-plugins-core
 Summary:    Core Plugins for DNF
 Group:      System Environment/Base
-BuildRequires:  python-dnf >= %{dnf_version}
+BuildRequires:  python-dnf >= %{dnf_lowest_compatible}
+BuildRequires:  python-dnf < %{dnf_not_compatible}
```

Comment:
It is a build requirement so if you break API in dnf 1.0.0, you would properly change dnf-plugin-core to use the new API and build a new verison with
BuildRequires:  python-dnf >= 1.0 and release it togher with dnf-1.0.0
You can't push a new version of dnf into a repo there breaks requirments of an installed version of dnf-plugins-core, then it would not be able to update.